server/handlers/shared_validators.js

```diff
@@ -245,3 +245,16 @@ exports.availabilityPackagingBodyValidator = body("packaging")
   .bail()
   .trim()
   .escape();
+
+// stats validators
+exports.weekQueryValidator = query("week").optional().notEmpty().bail().isInt();
+
+exports.yearQueryValidator = query("year").optional().notEmpty().bail().isInt();
+
+exports.startWeekQueryValidator = query("startWeek").notEmpty().bail().isInt();
+
+exports.endWeekQueryValidator = query("endWeek").notEmpty().bail().isInt();
+
+exports.startYearQueryValidator = query("startYear").notEmpty().bail().isInt();
+
+exports.endYearQueryValidator = query("endYear").notEmpty().bail().isInt();
```

Comment:
The maximum value for the week should be 52 and minimum should be 0 or 1 , you can add this check inside isInt()
Also is convenient to add a minimum & maximum value for the year too

Reply:
it depends on the year for the maximum. so I didn't put the check